Source/MLXNN/Activations.swift

```diff
@@ -137,7 +139,8 @@ public func softSign(_ x: MLXArray) -> MLXArray {
 /// - <doc:activations>
 /// - ``CELU``
 public func celu(_ x: MLXArray, alpha: Float = 1.0) -> MLXArray {
-    maximum(x, 0.0) + alpha * (exp(minimum(x, 0.0) / alpha) - 1)
+    let alpha = alpha.asMLXArray(dtype: x.dtype)
+    return compiledCelu(x, alpha)
```

Comment:
Note: here I hoist the conversion to MLXArray so that we can pass the alpha value into the compiled code.